0800-appendix-198116-waterfall.Rmd

```diff
@@ -15,58 +15,58 @@ source('scripts/tables.R')
 
 <br>
 
-## Background {.unnumbered}
+During 2022 field surveys, three previously un-assessed stream crossings located upstream of the upper most railway culverts (PSCIS 124423) were surveyed for fish passage (PSCIS 198115 on 9th avenue, 198116 on Highway 16/10th avenue and 198117 on 13th avenue). Downstream of these three crossings the stream flows through six more road/rail crossings before it's confluence with Station Creek. A small bridge (modelled crossing 24704553 - a.k.a Mile 48) is located under the CN Railway approximately 875m upstream of the highway and at the time of reporting, there was an un-assessed crossing (1803294) on the subject tributary just upstream of it's confluence with Waterfall Creek. Site surveys downstream in Waterfall Creek in 2021 along with review of aerial imagery indicate that modelled crossing 1803294 likely does not exist as the stream appears to run adjacent to the railway at this location and not cross the road.  The location of modeled crossing 1803294 would need to be accessed from the west by crossing the CN Railway or from the east via a private road located behind gates to the New Hazelton sewage treatment facility.
+
+<br>
+
+Although the habitat confirmation assessment in this memo is detailed as for crossing 198116, the assessment conducted also covered areas downstream of 198117 and upstream of crossing 198115 so has significance for informing remediation of those culverts as well. The crossing locations were prioritized for follow up after field reconnaissance indicated that `r fpr::fpr_my_pscis_info()` was mapped incorrectly in the BC Freshwater Atlas stream network and likely contained significant quantities of high value habitat for coho spawning and rearing.
+
+<br>
 
-At crossing `r as.character(my_site)`, `r fpr::fpr_my_pscis_info()` is a
-first order stream. Upstream of crossing `r as.character(my_site)`, cutthroat trout and dolly varden have previously been recorded
-[@moe2020KnownBC; @norris2020bcfishobs].  
+
+As the subject tributary to Waterfall Creek was not mapped as present at the location of these crossings, at the time of reporting, there were no crossings modelled as present in `bcfishpass` and so the software generates no fish habitat modelling data. PSCIS crossing `r as.character(my_site2)` is located on `r fpr_my_pscis_info(site = my_site2, col_pull = road_name)`, just downstream of 198116. PSCIS crossing `r as.character(my_site3)` is located on `r fpr_my_pscis_info(site = my_site3, col_pull = road_name)`, approximately 370m upstream of 198116. Crossing `r as.character(my_site)` on Hiighway 16/10th avenue is the responsibility of the Ministry of Transportation and Infrastructure while crossings  `r as.character(my_site2)` and  `r as.character(my_site3)` are the responsibility of the District of New Hazelton.
```

Comment:
There's two i's in highway in the last sentence.

Reply:
might be a "collaborator" thing.  will be good to experiment in the future.   Its odd right now because usually I can review my own PR and then the merge button will go green so I can do it. Now it is telling me you have "approved these changes" but no merge allowed and it says I can't review my own PR. interestante 

![image](https://user-images.githubusercontent.com/46538103/211049628-1df09559-44f7-4d7e-9662-a4255ae3de86.png)